src/main/java/com/danielagapov/spawn/Models/FriendRequests.java

```diff
@@ -0,0 +1,34 @@
+package com.danielagapov.spawn.Models;
+
+import com.danielagapov.spawn.Models.User;
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.io.Serializable;
+import java.util.UUID;
+
+@Entity
+@Table(
+        name = "friend_requests",
+        uniqueConstraints = @UniqueConstraint(columnNames = {"sender_id", "receiver_id"})
```

Comment:
Great example entity where this unique constraint is very useful. Good job. In other entities, I'm not sure if they're as necessary, but they're fine to keep for principal's sake.